src/ArrayCache.php

```diff
@@ -203,9 +210,11 @@ private function validateKey($key): void
 
     /**
      * @param array $keys
+     * @psalm-assert string[] $keys
```

Comment:
Method paramater `$keys` is array of mixed.

Reply:
This is an assert annotation that means the method can accept any type of value, but it validates to a certain type